core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -13,7 +13,31 @@
 #http.enabled=true
 
 # IP address and port of HTTP server
-#http.bind.to=0.0.0.0:9000
+#http.net.bind.to=0.0.0.0:9000
```

Comment:
This has high chances to be changed from default in some installations already. Can we make it backward compatible or avoid changing?

Reply:
> All old property names are also supported.